app/submitter/converter_v2.py

```diff
@@ -49,15 +49,15 @@ def convert_answers_v2(
     Returns:
         Data payload
     """
-    metadata = questionnaire_store.metadata
+    metadata = questionnaire_store.data_stores.metadata
     if not metadata:
         raise NoMetadataException
 
-    response_metadata = questionnaire_store.response_metadata
-    answer_store = questionnaire_store.answer_store
-    list_store = questionnaire_store.list_store
-    progress_store = questionnaire_store.progress_store
-    supplementary_data_store = questionnaire_store.supplementary_data_store
+    response_metadata = questionnaire_store.data_stores.response_metadata
+    answer_store = questionnaire_store.data_stores.answer_store
+    list_store = questionnaire_store.data_stores.list_store
+    progress_store = questionnaire_store.data_stores.progress_store
+    supplementary_data_store = questionnaire_store.data_stores.supplementary_data_store
 
     survey_id = schema.json["survey_id"]
```

Comment:
on line 115, `get_payload_data` still has all the params, but it could be changed to just having `data_stores` and a large number of places calling it then don't need to pass each parameter from `questionnaire_store.data_stores` one by one